rls-analysis/src/raw.rs

```diff
@@ -50,9 +50,9 @@ impl Crate {
 
 /// Reads raw analysis data for non-blacklisted crates from files in directories
 /// pointed by `loader`.
-pub fn read_analysis_from_files<L: AnalysisLoader>(
+pub fn read_analysis_from_files<L: AnalysisLoader, S: std::hash::BuildHasher + Default>(
     loader: &L,
-    crate_timestamps: HashMap<PathBuf, SystemTime>,
+    crate_timestamps: HashMap<PathBuf, SystemTime, S>,
```

Comment:
Why do we explicitly specify hasher for the HashMap? I believe this shouldn't be necessary

Reply:
so this was auto-recommended by clippy to me, but i took it out in the latest commit

https://rust-lang.github.io/rust-clippy/master/#implicit_hasher